frontend/src/components/result.js

```diff
@@ -175,9 +215,9 @@ export class ResultView extends React.Component {
     return (
       <React.Fragment>
         {this.state.testResult &&
-        <Tabs activeKey={this.state.artifactsTabKey} onSelect={this.handleArtifactsTabClick}>
+        <Tabs activeKey={activeTab} onSelect={this.onTabSelect} isBox>
```

Comment:
Can we also add `isBox` to the `Tabs` on the Jenkins Job analysis view?
https://github.com/ibutsu/ibutsu-server/blob/master/frontend/src/views/jenkinsjobanalysis.js#L210

Reply:
Do you want me to implement the tab remembering on this page too?

Reply:
If you'd like to. We can always do it as a follow-up.

Reply:
Might as well :-)

Reply:
Done :-)